Add format radio option description to control docs (#125)
eclipsesource/jsonforms2-website: src/pages/docs/controls.mdx

In the following example each element will be labeled with its name instead of its message because we provide the name prop.

<ExampleWithArrayLabel />

### Radio groups

**`format: 'radio'`**
```
options: {
  format: 'radio'
}
```

> This options is used to display an enum as a radio group.

Reviewer: ```suggestion
Use the `format: 'radio'` option to display an enum as a radio group.
```
